apim-operator/pkg/controller/api/api_controller.go

```diff
@@ -687,6 +694,36 @@ func dockerfileHandler(r *ReconcileAPI) (*corev1.ConfigMap, error) {
 	} else if err != nil {
 		return dockerfileConfmap, err
 	}
-	
+
 	return dockerfileConfmap, err
 }
+
+func policyHandler(r *ReconcileAPI) error {
+	//Check if policy configmap is available
+	foundmapc := &corev1.ConfigMap{}
+	err := r.client.Get(context.TODO(), types.NamespacedName{Name: "policy-configmap", Namespace: "wso2-system"}, foundmapc)
```

Comment:
Shall we move the policy-configmap and wso2-system to constants with proper name?